src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsExampleHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Text.Json;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen
+{
+    public static class XmlCommentsExampleHelper
+    {
+        public static IOpenApiAny Create(
+            SchemaRepository schemaRepository,
+            OpenApiSchema schema,
+            string exampleString)
+        {
+            var isStringType =
```

Comment:
Should this check if schema is null before calling `.ResolveType()`?

I don't think it happens in the intended use but I have a version of `XmlCommentsOperationFilter` which handles response tags like:

```xml
<response code="400" contentType="application/problem+json" type="Library.ProblemDetails">
  <example>{ "message": "There was a problem", "statusCode": 400, errors: [] }</example>
  <description>Bad Request - data validation error</description>
</response>
```

and with my operation filter calling this method, my unit tests fail with a null ref on schema here when the type cannot be found.

Reply:
@bbarry yes, it should check if `schema` is null before calling `.ResolveType()`. Thanks for finding that issue. I have updated the PR.